stl/inc/sstream

```diff
@@ -326,6 +439,41 @@ protected:
         _Mystate = _State;
     }
 
+#if _HAS_CXX20
+    void _Init_inplace(_Mystr&& _Str, int _State) {
+        if ((_State & _Constant) && !(_State & _Noread)) { // readonly, copy the string
+            _Init(_Str.c_str(), _Str.size(), _State);
+        } else { // writeable, construct in place
+            auto _Buffer = _Str._Release_to_buffer();
+            _Init_inplace(_STD get<0>(_Buffer), _STD get<1>(_Buffer), _STD get<2>(_Buffer), _State);
+        }
+    }
+
+    // Call this method only when writeable

```

Comment:
Can/should this be expressed as an `_STL_INTERNAL_CHECK` (which is active in our test suite, but not for users)?